packages/backend/src/managers/modelsManager.ts

```diff
@@ -0,0 +1,26 @@
+import type { LocalModelInfo } from '@shared/src/models/ILocalModelInfo';
+import fs from 'fs';
+import * as path from 'node:path';
+
+export class ModelsManager {
+  constructor(private appUserDirectory: string) {}
+
+  getLocalModels(): LocalModelInfo[] {
```

Comment:
just a question.
It seems it's a lazy initialization and without cache, do we need to read the files each time we're called or you're planning to read it eagerly or have a cache

Reply:
We could watch the folder for changes

Reply:
Good point, I created #116